analyzer/runtime/extract.go

```diff
@@ -364,15 +364,42 @@ func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []nodeapi.Runtime
 					blockTransactionData.Body = body
 					amount = body.Amount.Amount
 					if body.To != nil {
-						// Beware, this is the address of an account in the consensus
-						// layer, not an account in the runtime that generated this event.
+						// This is the address of an account in the consensus
+						// layer, not an account in the runtime that generated this tx.
 						// We do not register it as a preimage.
 						if to, err = uncategorized.StringifySdkAddress(body.To); err != nil {
 							return fmt.Errorf("to: %w", err)
 						}
 					}
 					return nil
 				},
+				ConsensusAccountsDelegate: func(body *consensusaccounts.Delegate) error {
```

Comment:
I tnink no need, since the appropriate `account` burn/mint events will be emitted by the SDK on the delegation and once the funds are reclaimed. And those are already handled in https://github.com/oasisprotocol/nexus/blob/fb5d2fb1ed3836f0bf15d2edd076fd8b255ba9e6/analyzer/runtime/accounts.go#L18-L31

Reply:
Correct. (With a small caveat -- there's an additional Transfer involved.) I was also unsure of this when I started this PR, and it took me some spelunking to find the explanation before I was comfortable writing the code. Good question, Andy, and worth spelling out the answer for future maintainers. See the new "LESSON" comment in the code for an explanation of events.

Reply:
thanks for adding the comnent! v helpful